README.md

```diff
@@ -30,8 +30,8 @@ Firstly, uninstall the official release of the extension and reload VS Code if y
 cd sway-vscode-plugin
 npm i
 git checkout <branch>
-vsce package  # Generates a .vsix file
-code --install-extension sway-vscode-extension-<version>.vsix
+vsce package  # Generates a .vsix file, the version will be included in the file name
```

Comment:
Hmm. I would prefer either an automatic extraction of the version with a shell command, or just hard-coding it into the readme and making sure it's updated in lockstep with `package.json`'s version.

Reply:
Important side-note: I had already run `npm i` when going through this myself, and the `vsce` command did not register until I used `npm install -g vsce` (this command also seems to be everywhere this topic is mentioned)

Reply:
Oh, that's because when running `npm i` `vsce` is installed into `node_modules/bin`. I have `./node_modules/bin` in my `PATH`. I think this is standard practice? If not, then adding an explicit path to `vsce` inside `node_modules` is better than installing it globally without a fixed version.